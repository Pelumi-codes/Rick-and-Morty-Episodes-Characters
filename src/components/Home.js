import React from "react";
import { Box, Text } from "@chakra-ui/react";
const Home = () => {
  return (
    <Box>
      <Text
        px="20px"
        bgColor="gray.200"
        width="90px"
        mt="10px"
        ml="30px"
        py="10px"
        borderRadius="5px"
        cursor="pointer"
      >
        Home
      </Text>
    </Box>
  );
};

export default Home;
