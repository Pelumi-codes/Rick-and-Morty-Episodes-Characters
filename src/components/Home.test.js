import React from "react";
import renderer from "react-test-renderer";

import Home from "./Home";
it("renders home button", () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
