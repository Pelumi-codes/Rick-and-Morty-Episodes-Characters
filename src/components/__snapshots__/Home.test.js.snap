// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home button 1`] = `
<div
  className="css-0"
>
  <p
    className="chakra-text css-134pyzd"
  >
    Home
  </p>
</div>
`;
